.github/workflows/release.yaml

```diff
@@ -278,10 +278,13 @@ jobs:
     - name: Bump Version File
       if: inputs.release_candidate == false
       run: |
+        # Trigger a bump of the version file
         set -e
         git checkout ${GITHUB_REF#refs/heads/}
         v="$(go run ./api/version/generate bump-version)"
         echo "$v" > VERSION
+        # Trigger a bump of any potential files that depend on a new version
+        make generate
         git add VERSION
```

Comment:
I guess you'll need to add the generated markdown files here as well

Reply:
Thanks for the correction, changed the add to --all since I dont know in advance which files changed